content/en/users/dev-env/binder/_index.md

```diff
@@ -3,9 +3,116 @@ title: Binder
 type: docs
 weight: 20
 description: >
-  Custom computing environments for EGI Notebooks
+  Custom reproducible computing environments for notebooks
 ---
 
 ## What is it?
 
-Repeatable, custom computing environments for notebooks
+Binder allows the re-creation of a custom computing environment for reproducible
+execution of notebooks (and potentially many other types of applications). Users
+who create their own notebooks in the [EGI Notebooks](https://notebooks.egi.eu)
```

Comment:
Shouldn't this link to the Notebooks users doc instead (that page contains a link to the actual service, instead)?

Reply:
yes, this was changed with Gergely. But I do agree it makes sense to move it to the docs instead. Fixing it